hls4ml/backends/oneapi/oneapi_backend.py

```diff
@@ -175,6 +175,15 @@ def compile(self, model):
         try:
             subprocess.run('which icpx', shell=True, cwd=builddir, check=True)
         except subprocess.CalledProcessError:
+            try:
```

Comment:
Do we still need this try - except block if the `pytest` import only happens when it's available?